### PR TITLE
docs: update Typography doc

### PR DESCRIPTION
**Pull request by ohhoney1**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

#28991 was covered by #28821 .

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | update Typography doc |
| 🇨🇳 Chinese | 更新 Typography 文档 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/index.en-US.md](https://github.com/ohhoney1/ant-design/blob/update_Typography_doc/components/typography/index.en-US.md)
[View rendered components/typography/index.zh-CN.md](https://github.com/ohhoney1/ant-design/blob/update_Typography_doc/components/typography/index.zh-CN.md)

### CODE REVIEW

File: components/typography/index.zh-CN.md [hunk 1]
```diff
@@ -117,8 +118,8 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | ---------- | -------------------- | -------------------- | ------ | ------ |
 | expandable | 是否可展开           | boolean              | -      |        |
 | rows       | 最多显示的行数       | number               | -      |        |
-| suffix     | 自定义省略内容后缀   | ReactNode            | -      |        |
-| symbol     | 自定义省略符号       | ReactNode            | `...`  |        |
+| suffix     | 自定义省略内容后缀   | string               | -      |        |
+| symbol     | 自定义展开符号       | ReactNode            | `展开` |        |
```
Comment (xrkffgg) [inline components/typography/index.zh-CN.md:122]: 是省略号啊。

File: components/typography/index.zh-CN.md [hunk 2]
```diff
@@ -117,8 +118,8 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | ---------- | -------------------- | -------------------- | ------ | ------ |
 | expandable | 是否可展开           | boolean              | -      |        |
 | rows       | 最多显示的行数       | number               | -      |        |
-| suffix     | 自定义省略内容后缀   | ReactNode            | -      |        |
-| symbol     | 自定义省略符号       | ReactNode            | `...`  |        |
+| suffix     | 自定义省略内容后缀   | string               | -      |        |
+| symbol     | 自定义省略符号       | ReactNode            | `展开` |        |
```
Comment (xrkffgg) [inline components/typography/index.zh-CN.md:122]: 为啥要改成 展开？

Comment (ohhoney1) [reply components/typography/index.zh-CN.md:122]: https://github.com/ant-design/ant-design/blob/master/components/typography/Base.tsx#L346-L350
代码中这个 `symbol` 不是指大多人都以为是 `...`，高亮代码中的 `expandStr` 就是 国际化中 `expand`
或许可以考虑再加个可以修改 `...` 的功能；或者 symbol 换个名称，比如 expandNode

Comment (ohhoney1) [reply components/typography/index.zh-CN.md:122]: https://codepen.io/cnjs/pen/jOMgNzv

**Comment by xrkffgg** [comment]
@afc163 @zombieJ 
看了下代码，确实有问题。

https://github.com/ant-design/ant-design/blob/83904e56eae3efad33c20dc60604c86a579f1a84/components/typography/Base.tsx#L113

这个省略号默认不能修改，而 symbol 修改的内容是 “展开”

**Comment by zombieJ** [comment]
嗯，描述应该改掉。`自定义省略符号` 这个描述也不对。

**Comment by ohhoney1** [comment]
文案已改，看下这样描述清晰吗

File: components/typography/index.zh-CN.md [hunk 3]
```diff
@@ -117,8 +118,8 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | ---------- | -------------------- | -------------------- | ------ | ------ |
 | expandable | 是否可展开           | boolean              | -      |        |
 | rows       | 最多显示的行数       | number               | -      |        |
-| suffix     | 自定义省略内容后缀   | ReactNode            | -      |        |
-| symbol     | 自定义省略符号       | ReactNode            | `...`  |        |
+| suffix     | 自定义省略内容后缀   | string               | -      |        |
+| symbol     | 自定义展开描述文案   | ReactNode            | `展开` |        |
```
Comment (afc163) [inline components/typography/index.zh-CN.md:122]: 为啥是展开？默认应该是 `...` 没错的。

Comment (ohhoney1) [reply components/typography/index.zh-CN.md:122]: https://codepen.io/cnjs/pen/jOMgNzv
大佬也可以看看之前的 review 。不过这个 symbol 的释义确实让好多人误会

**Pull request merged**